README.md

```diff
@@ -50,6 +50,26 @@ Divvy allows users to enter expenses as they are incurred, and calculates the be
 
 ### Installing Dependencies (tbd)
 
+### Setting Up passport-facebook
+
+1. Sign up to be a [facebook developer](https://developers.facebook.com/).
+2. Click the "Add a New App" on your [apps page](https://developers.facebook.com/apps/).
+3. Choose a "Website" app and give it a name.
+4. Add a contact email and a category when prompted.
+5. Complete the captcha and wait for the app to be created.
+6. Under the "Tell us about your website" portion of the next page, enter:  
+     `http://localhost:3000/`
+7. Click the "Skip Quick Start" button.
+8. Click the "+ Add Product" button on the left side of the dashboard.
+9. Click the "Get Started" button next to "Facebook Login".  
+10. For "Valid OAuth redirect URIs", enter:  
+     `http://localhost:3000/auth/facebook/callback`
+11. Click "Save Changes" on the bottom right.
+12. Go back to the app dashboard by clicking the "Dashboard" menu item on the top left.
+13. If you haven't already, make a copy of `./server/config/.env.sample.js` as `./server/config/.env.js`.
+14. Copy the "App ID" and replace `FACEBOOK_ID`'s value with the copied value. Notice that the `.env.js` file is a javascript file. The value for each key in this file should be a string so make sure there are single quotes around the pasted in value.
```

Comment:
Add a 15th step similar to: While running grunt, navigate to Localhost:3000/auth/facebook and look at the response in your browser. A cookie from the server with a token value should be displayed.